hack/sdk-gen/README.md

```diff
@@ -0,0 +1,46 @@
+
+# Readme for Generating KFServing SDK
+
+The guide shows how to generate the openapi model and swagger.json file from kersering types using `openapi-gen` and generate KFServing python sdk for the python object models using `swagger-codegen`.
+
+### Install openapi-gen
+
+```
+cd ./vendor/k8s.io/code-generator/cmd/openapi-gen
+go install -v
+```
+
+### Generate the openapi model from KFServing types
+
+Update the following kative files to add `+k8s:openapi-gen=true` (This should be updated from knative).
+```
+vendor/github.com/knative/pkg/apis/condition_types.go
```

Comment:
hmm not sure if we should change files in vendor, next person who run `dep ensure` can overwrite all these changes.

Reply:
Yes, your point is absolutely correct! So I mentioned this in README file so that following contributor can do this manually before running the command, and the changes are not committed. In fact the best way to enhance this from knative side (may send a PR to update this later for knative). For now, we can just update that manually otherwise the knative part will be skipped taht will cause import errors.

Reply:
Send a [PR](https://github.com/knative/pkg/pull/510) to update in knative pkg, but now I think we need to update that before executing openapi-ge.